test/transpiler/decorators.test.ts

```diff
@@ -729,7 +729,7 @@ test("decorators with different property key types", () => {
       "string" = 30;
       @d1("string method")
       "string method"() {}
-      @d1(12e3)
+      @d1(12000)
```

Comment:
why?

Reply:
I think that fixed a broken test a few days ago lol